src/components/fx/hover.js

```diff
@@ -653,20 +662,31 @@ function createHoverText(hoverData, opts, gd) {
     // show the common label, if any, on the axis
     // never show a common label in array mode,
     // even if sometimes there could be one
-    var showCommonLabel = c0.distance <= opts.hoverdistance &&
-                          (hovermode === 'x' || hovermode === 'y');
+    var showCommonLabel = (
+        (t0 !== undefined) &&
```

Comment:
Related to https://github.com/plotly/plotly.js/pull/2379#discussion_r168913340
Previously `c0.distance <= opts.hoverdistance` was strangely being used as a proxy for which trace types should show the common label. I suspect that clause could be removed entirely at this point, in fact... But now that we're not using that, it's important to ensure that there *is* a common label to display, otherwise you can get that little empty arrow in some cases.

Reply:
Good to know. Thanks!